packages/cli/src/commands/org/create.ts

```diff
@@ -0,0 +1,50 @@
+import {Command, flags} from '@oclif/command';
+import {cli} from 'cli-ux';
+import {Config} from '../../lib/config/config';
+import {
+  Preconditions,
+  HasAccessToken,
+} from '../../lib/decorators/preconditions/';
+import {AuthenticatedClient} from '../../lib/platform/authenticatedClient';
+
+export default class OrgCreate extends Command {
+  static description = 'Create a new trial Coveo organization';
+
+  private configuration!: Config;
+
+  static flags = {
+    setDefaultOrganization: flags.boolean({
+      char: 'D',
+      default: false,
+      description: 'Set the newly created organization as the default one',
+    }),
+  };
+
+  static args = [
+    {
+      name: 'name',
+      description: 'The name to assign to the new organization.',
+      required: true,
+    },
+  ];
+
+  @Preconditions(HasAccessToken())
+  async run() {
+    const {args, flags} = this.parse(OrgCreate);
+
+    this.configuration = new Config(this.config.configDir, this.error);
+    const authenticatedClient = new AuthenticatedClient();
+    const newCreatedOrg = await authenticatedClient.createOrganization(
```

Comment:
Is this a long running operation ? Long running as in it takes multiple seconds for the end user ?

If so, you could look into using cli.action.start(), cli.action.pause() to provide nice visual feedback.

If it's "quick", then disregard this comment.

Reply:
In my experience, it's super fast.